api/swagger-spec/v1beta1.json

```diff
@@ -1312,65 +1312,6 @@
     ]
    },
    {
-    "path": "/apis/extensions/v1beta1/namespaces/{namespace}/ingress/{name}/status",
```

Comment:
I'm confused about how it was in the spec if it hadn't been cherrypicked yet. Are the verify scripts broken in the release branches?